docs/2.0. EDID to Receiver Caps mapping.md

```diff
@@ -1,84 +1,192 @@
-# AMWA BCP-XXX-YY: EDID to NMOS Receiver Caps Mapping
+# AMWA BCP-XXX-YY: EDID to NMOS Receiver Capabilities Mapping \[Work In Progress\]
 
-## Background
+{:.no_toc}
 
-Extended Display Identification Data (EDID) is a metadata format for sink devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA). This document is targeted against [E-EDID A2][E-EDID] files which consist of EDID 1.4 which is called Base EDID and an optional [CTA-861-G][CTA-861] Extension Block.
+* A markdown unordered list which will be replaced with the ToC, excluding the "Contents header" from above
 
-[AMWA BCP-004-01][BCP-004-01] describes a methodology for a similar purpose in relationship to [AMWA IS-04][IS-04].
+{:toc}
+
+Extended Display Identification Data (EDID) is a metadata format for an apparatus to describe its capabilities as a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA). This document is targeted against [E-EDID A2][E-EDID] which consist of EDID 1.4 (and covers EDID 1.3) and is refered to as _Base EDID_ and the [CTA-861-G][CTA-861] Extension Block imposed by HDMI. The information present in the EDID is subject to the requirements of the [Display Monitor Timing (DMT)][DMT] specification Version 1 Rev 12.
```

Comment:
> The information present in the EDID is subject to the requirements of the [Display Monitor Timing (DMT)][DMT] specification Version 1 Rev 12.

It follows from this sentence that DMT is a higher-level restriction for E-EDID. Will it be more precise to write that any DMT mentions in the text below refer DMT v1r12?

Reply:
The version is here and in the normative references which is always pointed at, on the hover of the link it shows the version. 🤔 

I think like this is fine